axelrod/strategies/axelrod_second.py

```diff
@@ -199,3 +202,158 @@ def strategy(self, opponent: Player) -> Action:
+    Has a variable 'AK' which increases each time a move is played whilst in state FD = 1. It has an initial value of 1.
+    Has a variable 'NK' which increases each time a move is played whilst in state FD = 2. It has an initial value of 1.
+    Has a variable 'AD' with an initial value of 5
+    Has a variable 'NO with an initial value of 0
+    When FD = 0:
+        If the opponent's last move (JA) was Cooperate, increase the value of C by 1
+        If Score (K) < 1.75 * Move Number (M), play opponent's last move
+        If (1.75 * M) <= K < (2.25 * M):
+            Calculate Probability P:
+            P = 0.25 + C/M - 0.25*S + (K - L)/100 + 4/M
+            Where L is the opponent's score so far
+            If Random (R) <= P:
+                Cooperate
+            Else:
+                Defect
+        If K >= (2.25 * M):
+            Calculate probability P:
+            P = 0.95 - (AD + NO - 5)/15 + 1/M**2 - J/4
+            Where J is the opponent's last move
+            If Random (R) <= P:
+                Cooperate
+            Else:
+                Set FD = 1
+                Defect
+    When FD = 1:
+        Set FD = 2
+        Set the variable 'AD':
+        AD = ((AD * AK) + 3 - (3 * J) + (2 * JA) - (JA * J)) / (AK + 1)
+        Where JA is the strategy's last move and J is the opponent's last move (C = 0, D = 1)
+        Increase the value of AK by 1
+        Cooperate
+    When FD = 2:
+        Set FD = 0
+        Set the variable 'NO':
+        NO = ((NO * NK) + 3 - (3 * J) + (2 * JA) - (JA * J) / (NK + 1)
+        Where JA the strategy's last move and J is the opponent's last move (C = 0, D = 1)
+        Increase the value of NK by 1
+        Cooperate
+    
+    Tranquilizer came in 27th place in Axelrod's second torunament.
+
+    Names:
+
+    - Craig Feathers: [Axelrod1980]_
+    - Tranquilizer: [Axelrod1980]_
+    """
+
+    name = 'Tranquilizer'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': {"game"},
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    # Initialised atributes
+    def __init__(self):
+        super().__init__()
+        self.FD = 0
+        self.consecutive_defections = 0
+        self.ratio_FD1 = 5
+        self.ratio_FD2 = 0
+        self.ratio_FD1_count = 1
+        self.ratio_FD2_count = 1
+        self.score = None
+        self.P = 1.1
+        self.current_score = 0
+        self.dict = {C: 0, D: 1}
+
+
+    def update_stateFD(self, opponent):  # Calculates the ratioFD values and P values, as well as sets the states of FD at the start of each turn
+
+        self.current_score = compute_final_score(zip(self.history, opponent.history))
+
+        if self.FD == 2:
+            self.FD = 0
+            self.ratio_FD2 = ((self.ratio_FD2 * self.ratio_FD2_count + 3 - 3 * self.dict[opponent.history[-1]]) + 2 * self.dict[
```

Comment:
This (and similar lines) are breaking PEP8 (limit to 80 characters) in quite a few places. This is mainly because this is such a mouthful. 

Try and fix it by modifying things so that you create the summation in multiple steps perhaps.